setup.py

```diff
@@ -8,6 +8,15 @@
 
 read = lambda filepath: codecs.open(filepath, 'r', 'utf-8').read()
 
+install_requires = [
+    'Django>=1.6',
+]
+
+# Needed for Python <2.7
```

Comment:
django-jenkins isn't being tested on Python 2.6, so it doesn't officially support Python 2.6.

This should either be removed, or you should add a new python2.6 tox env to this PR so we can make sure it works.